src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.DefaultItems.props

```diff
@@ -82,18 +82,22 @@ Copyright (c) .NET Foundation. All rights reserved.
 
   </ItemGroup>
 
-  <!-- C# implicit imports -->
+  <!-- Implicit imports -->
   <ItemGroup Condition=" '$(DisableImplicitNamespaceImports_DotNet)' != 'true'
-                      and '$(Language)' == 'C#'
                       and '$(TargetFrameworkIdentifier)' == '.NETCoreApp'
                       and $([MSBuild]::VersionGreaterThanOrEquals($(_TargetFrameworkVersionWithoutV), '6.0'))">
-    <Import Include="System" />
-    <Import Include="System.Collections.Generic" />
+
+    <!-- These imports are already defined as VB implicit imports, so only add them for C# here -->
+    <Import Include="System" Condition="'$(Language)' == 'C#'" />
+    <Import Include="System.Collections.Generic" Condition="'$(Language)' == 'C#'" />
+    <Import Include="System.Linq" Condition="'$(Language)' == 'C#'" />
+    <Import Include="System.Threading.Tasks" Condition="'$(Language)' == 'C#'" />
+    
+    <!-- These imports were not included by default for VB projects prior to .NET 6, so they apply to both languages -->
     <Import Include="System.IO" />
```

Comment:
Minor nit.

It would be nice to sort the `Import` items when generating the file now that we organize them logically. We used to preserve the alphabetical order as a side effect of how they are organized but it will no longer be the case after this change. I know it's rare for users to browse to this file, but the fact that it's a visible .cs file for C# projects in the obj directory makes me want to strive for more clean looking file.